[arrayfire] Build failure

Package: arrayfire[core,opencl]:x64-linux -> 3.8.0#4

**Host Environment**

- Host: x64-linux
- Compiler: GNU 9.4.0
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: adf5a7cee 2023-06-19 (2 days ago)

**To Reproduce**

`vcpkg install --enforce-port-checks --allow-unsupported`

**Failure logs**

```
-- Using cached arrayfire-arrayfire-d99887ae431fcd58168b653a1e69f027f04d5188.tar.gz.
-- Cleaning sources at /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/leander/vcpkg/downloads/arrayfire-arrayfire-d99887ae431fcd58168b653a1e69f027f04d5188.tar.gz
-- Applying patch build.patch
-- Applying patch Fix-constexpr-error-with-vs2019-with-half.patch
-- Applying patch fix-dependency-clfft.patch
-- Using source at /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean
-- Using cached arrayfire-threads-b666773940269179f19ef11c8f1eb77005e85d9a.tar.gz.
-- Cleaning sources at /home/leander/vcpkg/buildtrees/arrayfire/src/7005e85d9a-64aebcce4b.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/leander/vcpkg/downloads/arrayfire-threads-b666773940269179f19ef11c8f1eb77005e85d9a.tar.gz
-- Using source at /home/leander/vcpkg/buildtrees/arrayfire/src/7005e85d9a-64aebcce4b.clean
-- Using cached arrayfire-forge-1a0f0cb6371a8c8053ab5eb7cbe3039c95132389.tar.gz.
-- Cleaning sources at /home/leander/vcpkg/buildtrees/arrayfire/src/9c95132389-511398ace8.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/leander/vcpkg/downloads/arrayfire-forge-1a0f0cb6371a8c8053ab5eb7cbe3039c95132389.tar.gz
-- Using source at /home/leander/vcpkg/buildtrees/arrayfire/src/9c95132389-511398ace8.clean
-- Configuring x64-linux
-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /home/leander/vcpkg/downloads/tools/cmake-3.25.1-linux/cmake-3.25.1-linux-x86_64/bin/cmake --build . --config Debug --target install -- -v -j8
    Working Directory: /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg
    See logs for more information:
      /home/leander/vcpkg/buildtrees/arrayfire/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/arrayfire/portfile.cmake:93 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/home/leander/vcpkg/buildtrees/arrayfire/install-x64-linux-dbg-out.log</summary>

```
[1/459] cd /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl && /home/leander/vcpkg/downloads/tools/cmake-3.25.1-linux/cmake-3.25.1-linux-x86_64/bin/cmake -Dcfgdir= -P /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/third_party/CLBlast/tmp/CLBlast-ext-mkdirs.cmake && /home/leander/vcpkg/downloads/tools/cmake-3.25.1-linux/cmake-3.25.1-linux-x86_64/bin/cmake -E touch /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/third_party/CLBlast/src/CLBlast-ext-stamp/CLBlast-ext-mkdir
[2/459] /usr/bin/c++   -fPIC -g -fvisibility=hidden -std=c++14 -MD -MT CMakeFiles/bin2cpp.dir/CMakeModules/bin2cpp.cpp.o -MF CMakeFiles/bin2cpp.dir/CMakeModules/bin2cpp.cpp.o.d -o CMakeFiles/bin2cpp.dir/CMakeModules/bin2cpp.cpp.o -c /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/CMakeModules/bin2cpp.cpp
[3/459] : && /usr/bin/c++ -fPIC -g  CMakeFiles/bin2cpp.dir/CMakeModules/bin2cpp.cpp.o -o bin2cpp   && :
[4/459] cd /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/kernel && /home/leander/vcpkg/downloads/tools/cmake-3.25.1-linux/cmake-3.25.1-linux-x86_64/bin/cmake -E make_directory /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl/kernel_headers && /home/leander/vcpkg/downloads/tools/cmake-3.25.1-linux/cmake-3.25.1-linux-x86_64/bin/cmake -E echo \#include\ \</home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/kernel/ops.hpp\> >>"/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl/kernel_headers/ops.hpp" && /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/bin2cpp --file ops.cl --namespace opencl --output /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl/kernel_headers/ops.hpp --name ops_cl
[5/459] cd /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/kernel && /home/leander/vcpkg/downloads/tools/cmake-3.25.1-linux/cmake-3.25.1-linux-x86_64/bin/cmake -E make_directory /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl/kernel_headers && /home/leander/vcpkg/downloads/tools/cmake-3.25.1-linux/cmake-3.25.1-linux-x86_64/bin/cmake -E echo \#include\ \</home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/kernel/approx1.hpp\> >>"/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl/kernel_headers/approx1.hpp" && /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/bin2cpp --file approx1.cl --namespace opencl --output /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl/kernel_headers/approx1.hpp --name approx1_cl
[6/459] cd /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/kernel && /home/leander/vcpkg/downloads/tools/cmake-3.25.1-linux/cmake-3.25.1-linux-x86_64/bin/cmake -E make_directory /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl/kernel_headers && /home/leander/vcpkg/downloads/tools/cmake-3.25.1-linux/cmake-3.25.1-linux-x86_64/bin/cmake -E echo \#include\ \</home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/kernel/KParam.hpp\> >>"/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl/kernel_headers/KParam.hpp" && /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/bin2cpp --file KParam.hpp --namespace opencl --output /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl/kernel_headers/KParam.hpp --name KParam_hpp
...
Skipped 670 lines
...

                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Param.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Array.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/iota.hpp:11,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/iota.cpp:9:
/home/leander/vcpkg/vcpkg_installed/x64-linux/include/CL/cl2.hpp:18:151: note: #pragma message: cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.
   18 | #pragma message("cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.")
      |                                                                                                                                                       ^
[171/459] /usr/bin/c++ -DAFDLL -DAF_CACHE_KERNELS_TO_DISK -DAF_OPENCL -DAF_WITH_LOGGING -DBOOST_CHRONO_HEADER_ONLY -DBOOST_COMPUTE_HAVE_THREAD_LOCAL -DBOOST_COMPUTE_THREAD_SAFE -DCL_HPP_ENABLE_EXCEPTIONS -DCL_HPP_MINIMUM_OPENCL_VERSION=120 -DCL_HPP_TARGET_OPENCL_VERSION=120 -DCL_TARGET_OPENCL_VERSION=120 -DCL_USE_DEPRECATED_OPENCL_1_2_APIS -DOS_LNX -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DUSE_CLBLAST -Dafopencl_EXPORTS -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/magma -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../../include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg -I/home/leander/vcpkg/buildtrees/arrayfire/src/9c95132389-511398ace8.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/SYSTEM -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/extern/half/include -isystem /home/leander/vcpkg/vcpkg_installed/x64-linux/include -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include/CL/.. -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/third_party/CLBlast/include -fPIC -g -fPIC -fvisibility=hidden -Wno-ignored-attributes -Wall -std=c++14 -MD -MT src/backend/opencl/CMakeFiles/afopencl.dir/jit.cpp.o -MF src/backend/opencl/CMakeFiles/afopencl.dir/jit.cpp.o.d -o src/backend/opencl/CMakeFiles/afopencl.dir/jit.cpp.o -c /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/jit.cpp
In file included from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/cl2hpp.hpp:22,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Param.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Array.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/jit.cpp:10:
/home/leander/vcpkg/vcpkg_installed/x64-linux/include/CL/cl2.hpp:18:151: note: #pragma message: cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.
   18 | #pragma message("cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.")
      |                                                                                                                                                       ^
[172/459] /usr/bin/c++ -DAFDLL -DAF_CACHE_KERNELS_TO_DISK -DAF_OPENCL -DAF_WITH_LOGGING -DBOOST_CHRONO_HEADER_ONLY -DBOOST_COMPUTE_HAVE_THREAD_LOCAL -DBOOST_COMPUTE_THREAD_SAFE -DCL_HPP_ENABLE_EXCEPTIONS -DCL_HPP_MINIMUM_OPENCL_VERSION=120 -DCL_HPP_TARGET_OPENCL_VERSION=120 -DCL_TARGET_OPENCL_VERSION=120 -DCL_USE_DEPRECATED_OPENCL_1_2_APIS -DOS_LNX -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DUSE_CLBLAST -Dafopencl_EXPORTS -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/magma -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../../include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg -I/home/leander/vcpkg/buildtrees/arrayfire/src/9c95132389-511398ace8.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/SYSTEM -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/extern/half/include -isystem /home/leander/vcpkg/vcpkg_installed/x64-linux/include -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include/CL/.. -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/third_party/CLBlast/include -fPIC -g -fPIC -fvisibility=hidden -Wno-ignored-attributes -Wall -std=c++14 -MD -MT src/backend/opencl/CMakeFiles/afopencl.dir/lu.cpp.o -MF src/backend/opencl/CMakeFiles/afopencl.dir/lu.cpp.o.d -o src/backend/opencl/CMakeFiles/afopencl.dir/lu.cpp.o -c /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/lu.cpp
In file included from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/cl2hpp.hpp:22,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Param.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Array.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/lu.hpp:10,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/lu.cpp:11:
/home/leander/vcpkg/vcpkg_installed/x64-linux/include/CL/cl2.hpp:18:151: note: #pragma message: cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.
   18 | #pragma message("cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.")
      |                                                                                                                                                       ^
[173/459] /usr/bin/c++ -DAFDLL -DAF_CACHE_KERNELS_TO_DISK -DAF_OPENCL -DAF_WITH_LOGGING -DBOOST_CHRONO_HEADER_ONLY -DBOOST_COMPUTE_HAVE_THREAD_LOCAL -DBOOST_COMPUTE_THREAD_SAFE -DCL_HPP_ENABLE_EXCEPTIONS -DCL_HPP_MINIMUM_OPENCL_VERSION=120 -DCL_HPP_TARGET_OPENCL_VERSION=120 -DCL_TARGET_OPENCL_VERSION=120 -DCL_USE_DEPRECATED_OPENCL_1_2_APIS -DOS_LNX -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DUSE_CLBLAST -Dafopencl_EXPORTS -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/magma -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../../include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg -I/home/leander/vcpkg/buildtrees/arrayfire/src/9c95132389-511398ace8.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/SYSTEM -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/extern/half/include -isystem /home/leander/vcpkg/vcpkg_installed/x64-linux/include -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include/CL/.. -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/third_party/CLBlast/include -fPIC -g -fPIC -fvisibility=hidden -Wno-ignored-attributes -Wall -std=c++14 -MD -MT src/backend/opencl/CMakeFiles/afopencl.dir/math.cpp.o -MF src/backend/opencl/CMakeFiles/afopencl.dir/math.cpp.o.d -o src/backend/opencl/CMakeFiles/afopencl.dir/math.cpp.o -c /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/math.cpp
In file included from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/cl2hpp.hpp:22,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/types.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/backend.hpp:22,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/half.hpp:35,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/math.hpp:13,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/math.cpp:10:
/home/leander/vcpkg/vcpkg_installed/x64-linux/include/CL/cl2.hpp:18:151: note: #pragma message: cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.
   18 | #pragma message("cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.")
      |                                                                                                                                                       ^
[174/459] /usr/bin/c++ -DAFDLL -DAF_CACHE_KERNELS_TO_DISK -DAF_OPENCL -DAF_WITH_LOGGING -DBOOST_CHRONO_HEADER_ONLY -DBOOST_COMPUTE_HAVE_THREAD_LOCAL -DBOOST_COMPUTE_THREAD_SAFE -DCL_HPP_ENABLE_EXCEPTIONS -DCL_HPP_MINIMUM_OPENCL_VERSION=120 -DCL_HPP_TARGET_OPENCL_VERSION=120 -DCL_TARGET_OPENCL_VERSION=120 -DCL_USE_DEPRECATED_OPENCL_1_2_APIS -DOS_LNX -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DUSE_CLBLAST -Dafopencl_EXPORTS -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/magma -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../../include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg -I/home/leander/vcpkg/buildtrees/arrayfire/src/9c95132389-511398ace8.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/SYSTEM -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/extern/half/include -isystem /home/leander/vcpkg/vcpkg_installed/x64-linux/include -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include/CL/.. -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/third_party/CLBlast/include -fPIC -g -fPIC -fvisibility=hidden -Wno-ignored-attributes -Wall -std=c++14 -MD -MT src/backend/opencl/CMakeFiles/afopencl.dir/homography.cpp.o -MF src/backend/opencl/CMakeFiles/afopencl.dir/homography.cpp.o.d -o src/backend/opencl/CMakeFiles/afopencl.dir/homography.cpp.o -c /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/homography.cpp
FAILED: src/backend/opencl/CMakeFiles/afopencl.dir/homography.cpp.o 
/usr/bin/c++ -DAFDLL -DAF_CACHE_KERNELS_TO_DISK -DAF_OPENCL -DAF_WITH_LOGGING -DBOOST_CHRONO_HEADER_ONLY -DBOOST_COMPUTE_HAVE_THREAD_LOCAL -DBOOST_COMPUTE_THREAD_SAFE -DCL_HPP_ENABLE_EXCEPTIONS -DCL_HPP_MINIMUM_OPENCL_VERSION=120 -DCL_HPP_TARGET_OPENCL_VERSION=120 -DCL_TARGET_OPENCL_VERSION=120 -DCL_USE_DEPRECATED_OPENCL_1_2_APIS -DOS_LNX -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DUSE_CLBLAST -Dafopencl_EXPORTS -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/magma -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../../include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg -I/home/leander/vcpkg/buildtrees/arrayfire/src/9c95132389-511398ace8.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/SYSTEM -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/extern/half/include -isystem /home/leander/vcpkg/vcpkg_installed/x64-linux/include -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include/CL/.. -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/third_party/CLBlast/include -fPIC -g -fPIC -fvisibility=hidden -Wno-ignored-attributes -Wall -std=c++14 -MD -MT src/backend/opencl/CMakeFiles/afopencl.dir/homography.cpp.o -MF src/backend/opencl/CMakeFiles/afopencl.dir/homography.cpp.o.d -o src/backend/opencl/CMakeFiles/afopencl.dir/homography.cpp.o -c /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/homography.cpp
In file included from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/cl2hpp.hpp:22,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Param.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Array.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/homography.hpp:10,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/homography.cpp:10:
/home/leander/vcpkg/vcpkg_installed/x64-linux/include/CL/cl2.hpp:18:151: note: #pragma message: cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.
   18 | #pragma message("cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.")
      |                                                                                                                                                       ^
In file included from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/TemplateTypename.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/kernel_cache.hpp:17,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/kernel/homography.hpp:13,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/homography.cpp:13:
/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/kernel/homography.hpp: In function ‘std::array<opencl::Kernel, 5> opencl::kernel::getHomographyKernels(af_homography_type)’:
/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/kernel/homography.hpp:42:48: error: ‘DBL_EPSILON’ was not declared in this scope; did you mean ‘CL_DBL_EPSILON’?
   42 |         EPS, (std::is_same<T, double>::value ? DBL_EPSILON : FLT_EPSILON)));
      |                                                ^~~~~~~~~~~
/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/TemplateArg.hpp:29:61: note: in definition of macro ‘DefineKeyValue’
   29 | #define DefineKeyValue(key, arg) " -D " #key "=" + toString(arg)
      |                                                             ^~~
/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/kernel/homography.hpp:42:62: error: ‘FLT_EPSILON’ was not declared in this scope; did you mean ‘CL_FLT_EPSILON’?
   42 |         EPS, (std::is_same<T, double>::value ? DBL_EPSILON : FLT_EPSILON)));
      |                                                              ^~~~~~~~~~~
/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/TemplateArg.hpp:29:61: note: in definition of macro ‘DefineKeyValue’
   29 | #define DefineKeyValue(key, arg) " -D " #key "=" + toString(arg)
      |                                                             ^~~
/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/homography.cpp: In function ‘int opencl::homography(opencl::Array<T>&, const opencl::Array<float>&, const opencl::Array<float>&, const opencl::Array<float>&, const opencl::Array<float>&, const opencl::Array<float>&, af_homography_type, float, unsigned int)’:
/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/homography.cpp:42:65: error: ‘FLT_MAX’ was not declared in this scope
   42 |         err = createValueArray<float>(af::dim4(nsamples, iter), FLT_MAX);
      |                                                                 ^~~~~~~
/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/homography.cpp:14:1: note: ‘FLT_MAX’ is defined in header ‘<cfloat>’; did you forget to ‘#include <cfloat>’?
   13 | #include <kernel/homography.hpp>
  +++ |+#include <cfloat>
   14 | #include <af/dim4.hpp>
[175/459] /usr/bin/c++ -DAFDLL -DAF_CACHE_KERNELS_TO_DISK -DAF_OPENCL -DAF_WITH_LOGGING -DBOOST_CHRONO_HEADER_ONLY -DBOOST_COMPUTE_HAVE_THREAD_LOCAL -DBOOST_COMPUTE_THREAD_SAFE -DCL_HPP_ENABLE_EXCEPTIONS -DCL_HPP_MINIMUM_OPENCL_VERSION=120 -DCL_HPP_TARGET_OPENCL_VERSION=120 -DCL_TARGET_OPENCL_VERSION=120 -DCL_USE_DEPRECATED_OPENCL_1_2_APIS -DOS_LNX -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DUSE_CLBLAST -Dafopencl_EXPORTS -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/magma -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../../include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg -I/home/leander/vcpkg/buildtrees/arrayfire/src/9c95132389-511398ace8.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/SYSTEM -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/extern/half/include -isystem /home/leander/vcpkg/vcpkg_installed/x64-linux/include -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include/CL/.. -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/third_party/CLBlast/include -fPIC -g -fPIC -fvisibility=hidden -Wno-ignored-attributes -Wall -std=c++14 -MD -MT src/backend/opencl/CMakeFiles/afopencl.dir/match_template.cpp.o -MF src/backend/opencl/CMakeFiles/afopencl.dir/match_template.cpp.o.d -o src/backend/opencl/CMakeFiles/afopencl.dir/match_template.cpp.o -c /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/match_template.cpp
In file included from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/cl2hpp.hpp:22,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Param.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Array.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/match_template.hpp:10,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/match_template.cpp:10:
/home/leander/vcpkg/vcpkg_installed/x64-linux/include/CL/cl2.hpp:18:151: note: #pragma message: cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.
   18 | #pragma message("cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.")
      |                                                                                                                                                       ^
[176/459] /usr/bin/c++ -DAFDLL -DAF_CACHE_KERNELS_TO_DISK -DAF_OPENCL -DAF_WITH_LOGGING -DBOOST_CHRONO_HEADER_ONLY -DBOOST_COMPUTE_HAVE_THREAD_LOCAL -DBOOST_COMPUTE_THREAD_SAFE -DCL_HPP_ENABLE_EXCEPTIONS -DCL_HPP_MINIMUM_OPENCL_VERSION=120 -DCL_HPP_TARGET_OPENCL_VERSION=120 -DCL_TARGET_OPENCL_VERSION=120 -DCL_USE_DEPRECATED_OPENCL_1_2_APIS -DOS_LNX -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DUSE_CLBLAST -Dafopencl_EXPORTS -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/magma -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../../include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg -I/home/leander/vcpkg/buildtrees/arrayfire/src/9c95132389-511398ace8.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/SYSTEM -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/extern/half/include -isystem /home/leander/vcpkg/vcpkg_installed/x64-linux/include -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include/CL/.. -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/third_party/CLBlast/include -fPIC -g -fPIC -fvisibility=hidden -Wno-ignored-attributes -Wall -std=c++14 -MD -MT src/backend/opencl/CMakeFiles/afopencl.dir/join.cpp.o -MF src/backend/opencl/CMakeFiles/afopencl.dir/join.cpp.o.d -o src/backend/opencl/CMakeFiles/afopencl.dir/join.cpp.o -c /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/join.cpp
In file included from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/cl2hpp.hpp:22,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Param.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Array.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/join.cpp:10:
/home/leander/vcpkg/vcpkg_installed/x64-linux/include/CL/cl2.hpp:18:151: note: #pragma message: cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.
   18 | #pragma message("cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.")
      |                                                                                                                                                       ^
[177/459] /usr/bin/c++ -DAFDLL -DAF_CACHE_KERNELS_TO_DISK -DAF_OPENCL -DAF_WITH_LOGGING -DBOOST_CHRONO_HEADER_ONLY -DBOOST_COMPUTE_HAVE_THREAD_LOCAL -DBOOST_COMPUTE_THREAD_SAFE -DCL_HPP_ENABLE_EXCEPTIONS -DCL_HPP_MINIMUM_OPENCL_VERSION=120 -DCL_HPP_TARGET_OPENCL_VERSION=120 -DCL_TARGET_OPENCL_VERSION=120 -DCL_USE_DEPRECATED_OPENCL_1_2_APIS -DOS_LNX -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DUSE_CLBLAST -Dafopencl_EXPORTS -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/magma -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../../include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg -I/home/leander/vcpkg/buildtrees/arrayfire/src/9c95132389-511398ace8.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/SYSTEM -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/extern/half/include -isystem /home/leander/vcpkg/vcpkg_installed/x64-linux/include -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include/CL/.. -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/third_party/CLBlast/include -fPIC -g -fPIC -fvisibility=hidden -Wno-ignored-attributes -Wall -std=c++14 -MD -MT src/backend/opencl/CMakeFiles/afopencl.dir/ireduce.cpp.o -MF src/backend/opencl/CMakeFiles/afopencl.dir/ireduce.cpp.o.d -o src/backend/opencl/CMakeFiles/afopencl.dir/ireduce.cpp.o -c /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/ireduce.cpp
In file included from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/cl2hpp.hpp:22,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Param.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Array.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/ireduce.hpp:10,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/ireduce.cpp:9:
/home/leander/vcpkg/vcpkg_installed/x64-linux/include/CL/cl2.hpp:18:151: note: #pragma message: cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.
   18 | #pragma message("cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.")
      |                                                                                                                                                       ^
[178/459] /usr/bin/c++ -DAFDLL -DAF_CACHE_KERNELS_TO_DISK -DAF_OPENCL -DAF_WITH_LOGGING -DBOOST_CHRONO_HEADER_ONLY -DBOOST_COMPUTE_HAVE_THREAD_LOCAL -DBOOST_COMPUTE_THREAD_SAFE -DCL_HPP_ENABLE_EXCEPTIONS -DCL_HPP_MINIMUM_OPENCL_VERSION=120 -DCL_HPP_TARGET_OPENCL_VERSION=120 -DCL_TARGET_OPENCL_VERSION=120 -DCL_USE_DEPRECATED_OPENCL_1_2_APIS -DOS_LNX -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DUSE_CLBLAST -Dafopencl_EXPORTS -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/magma -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../../include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg -I/home/leander/vcpkg/buildtrees/arrayfire/src/9c95132389-511398ace8.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/SYSTEM -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/extern/half/include -isystem /home/leander/vcpkg/vcpkg_installed/x64-linux/include -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include/CL/.. -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/third_party/CLBlast/include -fPIC -g -fPIC -fvisibility=hidden -Wno-ignored-attributes -Wall -std=c++14 -MD -MT src/backend/opencl/CMakeFiles/afopencl.dir/meanshift.cpp.o -MF src/backend/opencl/CMakeFiles/afopencl.dir/meanshift.cpp.o.d -o src/backend/opencl/CMakeFiles/afopencl.dir/meanshift.cpp.o -c /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/meanshift.cpp
In file included from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/cl2hpp.hpp:22,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Param.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Array.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/meanshift.cpp:10:
/home/leander/vcpkg/vcpkg_installed/x64-linux/include/CL/cl2.hpp:18:151: note: #pragma message: cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.
   18 | #pragma message("cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.")
      |                                                                                                                                                       ^
[179/459] /usr/bin/c++ -DAFDLL -DAF_CACHE_KERNELS_TO_DISK -DAF_OPENCL -DAF_WITH_LOGGING -DBOOST_CHRONO_HEADER_ONLY -DBOOST_COMPUTE_HAVE_THREAD_LOCAL -DBOOST_COMPUTE_THREAD_SAFE -DCL_HPP_ENABLE_EXCEPTIONS -DCL_HPP_MINIMUM_OPENCL_VERSION=120 -DCL_HPP_TARGET_OPENCL_VERSION=120 -DCL_TARGET_OPENCL_VERSION=120 -DCL_USE_DEPRECATED_OPENCL_1_2_APIS -DOS_LNX -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DUSE_CLBLAST -Dafopencl_EXPORTS -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/magma -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../../include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg -I/home/leander/vcpkg/buildtrees/arrayfire/src/9c95132389-511398ace8.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/SYSTEM -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/extern/half/include -isystem /home/leander/vcpkg/vcpkg_installed/x64-linux/include -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include/CL/.. -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/third_party/CLBlast/include -fPIC -g -fPIC -fvisibility=hidden -Wno-ignored-attributes -Wall -std=c++14 -MD -MT src/backend/opencl/CMakeFiles/afopencl.dir/mean.cpp.o -MF src/backend/opencl/CMakeFiles/afopencl.dir/mean.cpp.o.d -o src/backend/opencl/CMakeFiles/afopencl.dir/mean.cpp.o -c /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/mean.cpp
In file included from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/cl2hpp.hpp:22,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Param.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Array.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/mean.hpp:11,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/mean.cpp:10:
/home/leander/vcpkg/vcpkg_installed/x64-linux/include/CL/cl2.hpp:18:151: note: #pragma message: cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.
   18 | #pragma message("cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.")
      |                                                                                                                                                       ^
[180/459] /usr/bin/c++ -DAFDLL -DAF_CACHE_KERNELS_TO_DISK -DAF_OPENCL -DAF_WITH_LOGGING -DBOOST_CHRONO_HEADER_ONLY -DBOOST_COMPUTE_HAVE_THREAD_LOCAL -DBOOST_COMPUTE_THREAD_SAFE -DCL_HPP_ENABLE_EXCEPTIONS -DCL_HPP_MINIMUM_OPENCL_VERSION=120 -DCL_HPP_TARGET_OPENCL_VERSION=120 -DCL_TARGET_OPENCL_VERSION=120 -DCL_USE_DEPRECATED_OPENCL_1_2_APIS -DOS_LNX -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DUSE_CLBLAST -Dafopencl_EXPORTS -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/magma -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../../include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg -I/home/leander/vcpkg/buildtrees/arrayfire/src/9c95132389-511398ace8.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/SYSTEM -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/extern/half/include -isystem /home/leander/vcpkg/vcpkg_installed/x64-linux/include -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include/CL/.. -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/third_party/CLBlast/include -fPIC -g -fPIC -fvisibility=hidden -Wno-ignored-attributes -Wall -std=c++14 -MD -MT src/backend/opencl/CMakeFiles/afopencl.dir/lookup.cpp.o -MF src/backend/opencl/CMakeFiles/afopencl.dir/lookup.cpp.o.d -o src/backend/opencl/CMakeFiles/afopencl.dir/lookup.cpp.o -c /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/lookup.cpp
In file included from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/cl2hpp.hpp:22,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/Param.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/kernel/lookup.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/lookup.cpp:10:
/home/leander/vcpkg/vcpkg_installed/x64-linux/include/CL/cl2.hpp:18:151: note: #pragma message: cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.
   18 | #pragma message("cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.")
      |                                                                                                                                                       ^
[181/459] /usr/bin/c++ -DAFDLL -DAF_CACHE_KERNELS_TO_DISK -DAF_OPENCL -DAF_WITH_LOGGING -DBOOST_CHRONO_HEADER_ONLY -DBOOST_COMPUTE_HAVE_THREAD_LOCAL -DBOOST_COMPUTE_THREAD_SAFE -DCL_HPP_ENABLE_EXCEPTIONS -DCL_HPP_MINIMUM_OPENCL_VERSION=120 -DCL_HPP_TARGET_OPENCL_VERSION=120 -DCL_TARGET_OPENCL_VERSION=120 -DCL_USE_DEPRECATED_OPENCL_1_2_APIS -DOS_LNX -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DUSE_CLBLAST -Dafopencl_EXPORTS -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/src/backend/opencl -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/magma -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/../../../include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/api/c -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend -I/home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg -I/home/leander/vcpkg/buildtrees/arrayfire/src/9c95132389-511398ace8.clean/include -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/SYSTEM -I/home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/extern/half/include -isystem /home/leander/vcpkg/vcpkg_installed/x64-linux/include -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/include/CL/.. -isystem /home/leander/vcpkg/buildtrees/arrayfire/x64-linux-dbg/third_party/CLBlast/include -fPIC -g -fPIC -fvisibility=hidden -Wno-ignored-attributes -Wall -std=c++14 -MD -MT src/backend/opencl/CMakeFiles/afopencl.dir/max.cpp.o -MF src/backend/opencl/CMakeFiles/afopencl.dir/max.cpp.o.d -o src/backend/opencl/CMakeFiles/afopencl.dir/max.cpp.o -c /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/max.cpp
In file included from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/cl2hpp.hpp:22,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/types.hpp:12,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/backend.hpp:22,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/common/half.hpp:35,
                 from /home/leander/vcpkg/buildtrees/arrayfire/src/27f04d5188-e800439fec.clean/src/backend/opencl/max.cpp:10:
/home/leander/vcpkg/vcpkg_installed/x64-linux/include/CL/cl2.hpp:18:151: note: #pragma message: cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.
   18 | #pragma message("cl2.hpp has been renamed to opencl.hpp to make it clear that it supports all versions of OpenCL. Please include opencl.hpp directly.")
      |                                                                                                                                                       ^
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "test",
  "version": "1.1.1",
  "dependencies": [
    {
      "name": "arrayfire",
      "features": [
        "opencl"
      ],
      "default-features": false
    }
  ],
  "$todo": [
    "pipewire",
    "openscap",
    "dpdk"
  ]
}

```
</details>

--- a/ports/arrayfire/build.patch
+++ b/ports/arrayfire/build.patch
@@ -218,3 +218,50 @@ index d618ff2..9f51739 100644
        )
  
      set_target_properties(opencl_sort_by_key_${SBK_TYPE}
+diff --git a/src/backend/opencl/CMakeLists.txt b/src/backend/opencl/CMakeLists.txt
+index b543433..9b5923b 100644
+--- a/src/backend/opencl/CMakeLists.txt
++++ b/src/backend/opencl/CMakeLists.txt
+@@ -432,12 +432,9 @@ if(AF_OPENCL_BLAS_LIBRARY STREQUAL "clBLAS")
+     PRIVATE
+       clBLAS::clBLAS)
+ elseif(AF_OPENCL_BLAS_LIBRARY STREQUAL "CLBlast")
+-  include(build_CLBlast)
++  find_package(CLBlast CONFIG REQUIRED)
+   target_compile_definitions(afopencl PRIVATE USE_CLBLAST)
+-  target_link_libraries(afopencl
+-    PRIVATE
+-      CLBlast)
+-    add_dependencies(afopencl CLBlast-ext)
++  target_link_libraries(afopencl PRIVATE clblast)
+ endif()
+
+
+diff --git a/src/backend/opencl/homography.cpp b/src/backend/opencl/homography.cpp
+index 3b598b0..2815923 100644
+--- a/src/backend/opencl/homography.cpp
++++ b/src/backend/opencl/homography.cpp
+@@ -39,7 +39,7 @@ int homography(Array<T> &bestH, const Array<float> &x_src,
+             ::std::min(iter, static_cast<unsigned>(
+                                  log(1.f - LMEDSConfidence) /
+                                  log(1.f - pow(1.f - LMEDSOutlierRatio, 4.f))));
+-        err = createValueArray<float>(af::dim4(nsamples, iter), FLT_MAX);
++        err = createValueArray<float>(af::dim4(nsamples, iter), std::numeric_limits<float>::max());
+     } else {
+         // Avoid passing "null" cl_mem object to kernels
+         err = createEmptyArray<float>(af::dim4(1));
+diff --git a/src/backend/opencl/kernel/homography.hpp b/src/backend/opencl/kernel/homography.hpp
+index b84e599..90d0c80 100644
+--- a/src/backend/opencl/kernel/homography.hpp
++++ b/src/backend/opencl/kernel/homography.hpp
+@@ -39,7 +39,9 @@ std::array<Kernel, 5> getHomographyKernels(const af_homography_type htype) {
+     };
+     options.emplace_back(getTypeBuildDefinition<T>());
+     options.emplace_back(DefineKeyValue(
+-        EPS, (std::is_same<T, double>::value ? DBL_EPSILON : FLT_EPSILON)));
++        EPS, (std::is_same<T, double>::value
++                                 ? std::numeric_limits<double>::epsilon()
++                                 : std::numeric_limits<float>::epsilon())));
+     if (htype == AF_HOMOGRAPHY_RANSAC) {
+         options.emplace_back(DefineKey(RANSAC));
+     }

--- a/ports/arrayfire/vcpkg.json
+++ b/ports/arrayfire/vcpkg.json
@@ -43,7 +43,8 @@
       "description": "ArrayFire OpenCL backend",
       "dependencies": [
         "clfft",
-        "opencl"
+        "opencl",
+        "clblast"
       ]
     },
     "unified": {

--- a/ports/arrayfire/vcpkg.json
+++ b/ports/arrayfire/vcpkg.json
@@ -42,9 +42,9 @@
     "opencl": {
       "description": "ArrayFire OpenCL backend",
       "dependencies": [
+        "clblast",
         "clfft",
-        "opencl",
-        "clblast"
+        "opencl"
       ]
     },
     "unified": {

--- a/ports/arrayfire/vcpkg.json
+++ b/ports/arrayfire/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "arrayfire",
   "version-semver": "3.8.0",
-  "port-version": 4,
+  "port-version": 5,
   "description": "ArrayFire is a general-purpose library that simplifies the process of developing software that targets parallel and massively-parallel architectures including CPUs, GPUs, and other hardware acceleration devices.",
   "homepage": "https://github.com/arrayfire/arrayfire",
   "license": "BSD-3-Clause",

--- a/ports/clblast/fix_install_path.patch
+++ b/ports/clblast/fix_install_path.patch
@@ -1,0 +1,16 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 2d28050..e2a060d 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -347,7 +347,10 @@ endif()
+ # Installs the library
+ include(GNUInstallDirs)
+ 
+-install(TARGETS clblast EXPORT CLBlast DESTINATION ${CMAKE_INSTALL_LIBDIR})
++install(TARGETS clblast EXPORT CLBlast 
++        ARCHIVE DESTINATION lib
++        LIBRARY DESTINATION lib
++        RUNTIME DESTINATION bin)
+ install(FILES include/clblast_half.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+ if(OPENCL)
+   install(FILES include/clblast.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})

--- a/ports/clblast/portfile.cmake
+++ b/ports/clblast/portfile.cmake
@@ -4,6 +4,8 @@ vcpkg_from_github(
     REF 1.5.2
     SHA512 6693704321bb7623a632ebfc71dcf07bbe4ba6c6f03a2ecf52bc10b401ae546bf82cdd3f6cc28aa9ea10f40dc7b2e86a6530f32cfbd522e24d4cf6a75c8c1100
     HEAD_REF master
+    PATCHES
+        fix_install_path.patch
 )
 
 vcpkg_cmake_configure(
@@ -13,16 +15,6 @@ vcpkg_cmake_configure(
 )
 
 vcpkg_cmake_install()
-
-if(VCPKG_TARGET_IS_WINDOWS AND VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
-    file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/bin")
-    file(RENAME "${CURRENT_PACKAGES_DIR}/lib/clblast.dll" "${CURRENT_PACKAGES_DIR}/bin/clblast.dll")
-    
-    if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
-        file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/debug/bin")
-        file(RENAME "${CURRENT_PACKAGES_DIR}/debug/lib/clblast.dll" "${CURRENT_PACKAGES_DIR}/debug/bin/clblast.dll")
-    endif()
-endif()
 
 vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/CLBLast)
 vcpkg_copy_pdbs()

--- a/ports/clblast/vcpkg.json
+++ b/ports/clblast/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "clblast",
   "version": "1.5.2",
-  "port-version": 2,
+  "port-version": 3,
   "description": "A modern, lightweight, performant and tunable OpenCL BLAS library written in C++11.",
   "homepage": "https://github.com/CNugteren/CLBlast",
   "license": "Apache-2.0",

--- a/versions/a-/arrayfire.json
+++ b/versions/a-/arrayfire.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "2019e0cc5d45cd1ec58b8a55b6ecc5d0d7aead60",
+      "version-semver": "3.8.0",
+      "port-version": 5
+    },
+    {
       "git-tree": "37adfc5544ac7301c3ff4a8e16d84804d45b566a",
       "version-semver": "3.8.0",
       "port-version": 4

--- a/versions/a-/arrayfire.json
+++ b/versions/a-/arrayfire.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "2019e0cc5d45cd1ec58b8a55b6ecc5d0d7aead60",
+      "git-tree": "4e9b4b24be953b97638224c3563759c7d8a86602",
       "version-semver": "3.8.0",
       "port-version": 5
     },

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -218,7 +218,7 @@
     },
     "arrayfire": {
       "baseline": "3.8.0",
-      "port-version": 4
+      "port-version": 5
     },
     "arrow": {
       "baseline": "12.0.1",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1558,7 +1558,7 @@
     },
     "clblast": {
       "baseline": "1.5.2",
-      "port-version": 2
+      "port-version": 3
     },
     "clfft": {
       "baseline": "2.12.2",

--- a/versions/c-/clblast.json
+++ b/versions/c-/clblast.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "5d3c066f06df46440c055d265aa2dee1c21a9971",
+      "version": "1.5.2",
+      "port-version": 3
+    },
+    {
       "git-tree": "f3fc8c351cba8b45e06d835b27101617a5ba74b0",
       "version": "1.5.2",
       "port-version": 2

